### DISCUSSION TITLE
Hd108 led strips?

### DISCUSSION THREAD

**Discussion by jhollis505** [awawa-dev/HyperHDR#446] [Q&A]
I'm honestly looking for the best (and probably most expensive) solution for hyperhdr. I was looking at sk6812 rgbw strips then saw that the setup might be out of my ballpark. I saw hd108 strips mentioned elsewhere and I'm wondering how those would do with hyperhdr. (As a side note, is the sk6812 setup up to date that's on the website?)

**Comment by awawa-dev** [comment]
Hi
hd108 seems to have nice 16bits per channel and https://github.com/Makuna/NeoPixelBus probably supports it in the last version (grb48). But nobody has tested it since it's new, we don't utilize 48bits benefits anyway and it's only rgb not rgbw. So sk6812 rgbw probably is still recommended solution, I'm personally using 96leds/meter cold version. I'm planning a new detailed tutorial in a month or so  (if my new Aliexpress order arrives intact on time but it's christmas time...), some things will be done completely differently utilizing new software/hardware features and we will break some neopixel protocol limitation too so keep watching pinned topics here or the blog.
